mmv1/third_party/terraform/tests/resource_firebase_extensions_instance_test.go.erb

```diff
@@ -0,0 +1,151 @@
+
+func testAccFirebaseExtensionsInstance_firebaseExtentionsInstanceResizeImageBefore(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_storage_bucket" "images" {
+  provider                    = google-beta
+  project                     = "%{project_id}"
+  name                        = "tf-test-bucket-id%{random_suffix}"
+  location                    = "US"
+  uniform_bucket_level_access = true
+
+  # Delete all objects when the bucket is deleted
+  force_destroy = true
+}
+
+resource "google_firebase_extensions_instance" "resize_image" {
+  provider = google-beta
+  project  = "%{project_id}"
+  instance_id = "tf-test-storage-resize-images%{random_suffix}"
+  config {
+    extension_ref = "firebase/storage-resize-images"
+    extension_version = "0.1.37"
+
+    # The following params apply to the firebase/storage-resize-images extension. 
+    # Different extensions may have different params
+    params = {
+      DELETE_ORIGINAL_FILE = false
+      MAKE_PUBLIC          = false
+      IMAGE_TYPE           = "jpeg"
+      IS_ANIMATED          = true
+      FUNCTION_MEMORY      = 1024
+      DO_BACKFILL          = false
+      IMG_SIZES            = "200x200"
+      IMG_BUCKET           = google_storage_bucket.images.name
+      LOCATION             = "%{location}"
+    }
+
+    system_params = {
+      "firebaseextensions.v1beta.function/maxInstances"               = 3000
+      "firebaseextensions.v1beta.function/memory"                     = 256
+      "firebaseextensions.v1beta.function/minInstances"               = 0
+      "firebaseextensions.v1beta.function/vpcConnectorEgressSettings" = "VPC_CONNECTOR_EGRESS_SETTINGS_UNSPECIFIED"
+    }
+
+    allowed_event_types = [
+      "firebase.extensions.storage-resize-images.v1.complete"
+    ]
+
+    eventarc_channel = "projects/%{project_id}/locations/%{location}/channels/firebase"
+  }
+}
+`, context)
+}
+
+func testAccFirebaseExtensionsInstance_firebaseExtentionsInstanceResizeImageAfter(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_storage_bucket" "images" {
+  provider                    = google-beta
+  project                     = "%{project_id}"
+  name                        = "tf-test-bucket-id%{random_suffix}"
+  location                    = "US"
+  uniform_bucket_level_access = true
+
+  # Delete all objects when the bucket is deleted
+  force_destroy = true
+}
+
+resource "google_firebase_extensions_instance" "resize_image" {
+  provider = google-beta
+  project  = "%{project_id}"
+  instance_id = "tf-test-storage-resize-images%{random_suffix}"
+  config {
+    extension_ref = "firebase/storage-resize-images"
+    extension_version = "0.1.37"
+
+    # The following params apply to the firebase/storage-resize-images extension. 
+    # Different extensions may have different params
+    params = {
+      # Changed params
+      DELETE_ORIGINAL_FILE = true
+      MAKE_PUBLIC          = true
+      IMAGE_TYPE           = true
```

Comment:
From the latest VCR tests it looks like this line needs to be changed to avoid this validation error from the API:

> googleapi: Error 400: Request had 1 validation errors: `params.IMAGE_TYPE`: contains one or more invalid options, each must be one of `jpeg`, `webp`, `png`, `tiff`, `gif`, `avif`, `false`.

Reply:
Fixed. Please don't run the next test yet. I'm figuring out what's going on with the service account permission error.